lib/reader.ts

```diff
@@ -644,21 +666,21 @@ class ParquetEnvelopeReader {
 /**
  * Decode a consecutive array of data using one of the parquet encodings
  */
-function decodeValues(type, encoding, cursor, count, opts) {
+function decodeValues(type: string, encoding: ParquetCodec, cursor: Cursor, count: number, opts: Options | {bitWidth: number}) {
   if (!(encoding in parquet_codec)) {
     throw 'invalid encoding: ' + encoding;
   }
 
-  return parquet_codec[encoding].decodeValues(type, cursor, count, opts);
+  return parquet_codec[encoding].decodeValues(type, cursor, count, opts as Options);
 }
 
 
-function decodeStatisticsValue(value, column) {
+function decodeStatisticsValue(value: any, column: ParquetField | Options) {
```

Comment:
Side note. Do not change.

There might be a way to use generics to solve the any problem here via the ParquetField, but it is not worth that work.